### PR TITLE
add ability to leave multiple groups

### PR DESCRIPTION
**Pull request by edewit**
fixes: #496

### CODE REVIEW

**Comment by jonkoops** [comment]
@ssilvert can we merge this one?

**Comment by ssilvert** [comment]
> @ssilvert can we merge this one?

I will retest this afternoon.  If it looks good I will merge it.

**Comment by ssilvert** [comment]
> @edewit Now it looks like ability to leave multiple groups is broken.
> 
> 1. Create Group A at the root.
> 2. Create Group B at the root.
> 3. User joins both A and B
> 4. Select `Group membership` to select both A and B
> 5. Click `Leave` at the top of the table
> 6. Only A will be removed

@edewit Looks like this still doesn't work.

File: src/user/UserGroups.tsx [hunk 1]
```diff
@@ -170,10 +172,12 @@ export const UserGroups = ({ user }: UserGroupsProps) => {
 
   const [toggleDeleteDialog, DeleteConfirm] = useConfirmDialog({
     titleKey: t("leaveGroup", {
-      name: selectedGroup?.name,
+      count: selectedGroups.length,
+      name: selectedGroups[0]?.name,
     }),
     messageKey: t("leaveGroupConfirmDialog", {
-      groupname: selectedGroup?.name,
+      count: selectedGroups.length,
```
Comment (mfrances17) [inline src/user/UserGroups.tsx:179]: i don't think the message changes are working as expected:
leave-group

**Review by mfrances17 — CHANGES_REQUESTED** [review]
i'm getting the same results as @ssilvert during testing. also some string issues on the confirm dialog based on these changes.

**Comment by edewit** [comment]
@mfrances17 @ssilvert sorry dropped the ball on this one... this time it's really fixed

**Review by jonkoops — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]
Looks good now.  Merging.

**Pull request merged**